src/entrypoints/core.js

```diff
@@ -66,11 +74,23 @@ function main() {
   }
   if (localStorage.tokens) {
     window.tokens = JSON.parse(localStorage.tokens);
-    window.hassConnection = init(null, window.tokens.access_token).catch((err) => {
-      if (err !== ERR_INVALID_AUTH) throw err;
+    if (window.tokens.expires === undefined) {
+      window.tokens.expires = Date.now() - 1;
+    }
+    if (Date.now() > window.tokens.expires) {
```

Comment:
We don't need to do this check if we will do this inside `home-assistant-js-websocket` before making a connection.